dev/com.ibm.ws.microprofile.faulttolerance.2.0/src/com/ibm/ws/microprofile/faulttolerance20/impl/ExecutorBuilderImpl20.java

```diff
@@ -34,9 +34,9 @@ public ExecutorBuilderImpl20(WSContextService contextService, PolicyExecutorProv
     @Override
     public <W> Executor<W> buildAsync(Class<?> asyncResultWrapperType) {
         if (asyncResultWrapperType == Future.class) {
-            return (Executor<W>) new AsyncFutureExecutor<Object>(retryPolicy, circuitBreakerPolicy, timeoutPolicy, fallbackPolicy, bulkheadPolicy, scheduledExecutorService, contextService, metricRecorder);
+            return (Executor<W>) new AsyncFutureExecutor<Object>(retryPolicy, circuitBreakerPolicy, timeoutPolicy, fallbackPolicy, bulkheadPolicy, scheduledExecutorService, contextService, metricRecorder, asyncRequestContext);
```

Comment:
Copyright date

Reply:
And here (https://github.com/OpenLiberty/open-liberty/pull/10172/commits/2558071a7e069a593629a86132eb6abf686d2174)